doc/account.md

```diff
@@ -44,16 +44,16 @@ Initiating an Account
 
 The first action for an account is `initiate`. This sets up a few pieces of data needed to properly generate and track an account. We generate a `regId`, `mfaKey`, and `passwordSalt`. An email address must be passed in. These are then passed back to the user to store on their own and will be used to generate their password.
 
-This will save the information to a file named after the hash of the `regId` in the `registrationDir` location. This registration file will be deleted upons successfully creating and account.
+This will save the information to a file named after the hash of the `regId` in the `registrationDir` location. This registration file will be deleted upons successfully creating an account.
```

Comment:
No mention of storage.  Storage should get documented as well, plus additional links scattered in to make sure people know we are not talking about a local disk.

Imagine someone only reading this paragraph instead of the whole doc.  Everyone skims, so try to make each sentence unambiguous and clear.